code/mac/apps/vscode.talon

```diff
@@ -0,0 +1,19 @@
+app: Code
+os: mac
+
+-
+# Navigation and commands
+run command: key(cmd-shift-p)
+
+action(user.ide_find_file):
```

Comment:
For things like this, where the implementation of a particular action is OS-specific, i think i'd prefer to break those out into separate files from the voice commands.

Similar to like this
https://github.com/knausj85/knausj_talon/blob/master/code/win/apps/chrome.talon

This might be a good time to consider creating a generic_ide.talon (similar to generic_broswer), and consolidate the voice commands between this and

https://github.com/knausj85/knausj_talon/blob/master/apps/jetbrains.talon


Thoughts?